src/modules/0.0.3/aws_acm_import/index.ts

```diff
@@ -0,0 +1,42 @@
+import { ImportCertificateCommandInput } from '@aws-sdk/client-acm';
+import { AWS, } from '../../../services/gateways/aws';
+import { Context, Crud, Mapper, Module, } from '../../interfaces'
+import { AwsAcmListModule } from '../aws_acm_list';
+import { CertificateImport } from './entity';
+import * as metadata from './module.json';
+
+
+export const AwsAcmImportModule: Module = new Module({
+  ...metadata,
+  mappers: {
+    certificateImport: new Mapper<CertificateImport>({
+      entity: CertificateImport,
+      entityId: (e: CertificateImport) => e.id?.toString() ?? '',
+      equals: () => true, // only database values
+      source: 'db',
+      cloud: new Crud({
+        create: async (es: CertificateImport[], ctx: Context) => {
+          const client = await ctx.getAwsClient() as AWS;
+          const textEncoder = new TextEncoder();
+          for (const e of es) {
+            const input: ImportCertificateCommandInput = {
+              Certificate: textEncoder.encode(e.body),
```

Comment:
Why is the certificate called `body` in the entity?

Reply:
oh I was just replicating the names they have in the interface. I can just change `body` to `certificate`.
![image](https://user-images.githubusercontent.com/5898639/167582647-386792c6-a404-4e62-a13e-fd02a8ba7ddd.png)